src/com/sibvisions/kitchensink/samples/containers/SplitPanelSample.java

```diff
@@ -58,10 +74,39 @@ public IPanel getContent()
 		splitPanelThird.setFirstComponent(splitPanelFirstSecond);
 		splitPanelThird.setSecondComponent(createCenteredLabel("Third", Tango.SKY_BLUE_1));
 		
+		IDataBook controlsBook = new MemDataBook();
+		controlsBook.getRowDefinition().addColumnDefinition(new ColumnDefinition("ORIENTATION", new BigDecimalDataType(createOrientationCellEditor())));
+		controlsBook.getRowDefinition().addColumnDefinition(new ColumnDefinition("ORIENTATION2", new BigDecimalDataType(createOrientationCellEditor())));
+		controlsBook.setName("CONTROLS");
+		controlsBook.open();
+		
+		controlsBook.insert(false);
+		controlsBook.setValue("ORIENTATION", BigDecimal.valueOf(splitPanelFirstSecond.getOrientation()));
+		controlsBook.setValue("ORIENTATION2", BigDecimal.valueOf(splitPanelThird.getOrientation()));
+		
+		controlsBook.eventValuesChanged().addListener(pDataRowEvent ->
+		{
+			IDataRow dataRow = pDataRowEvent.getChangedDataRow();
+			
+			splitPanelFirstSecond.setOrientation(((BigDecimal) dataRow.getValue("ORIENTATION")).intValue());
+			splitPanelThird.setOrientation(((BigDecimal) dataRow.getValue("ORIENTATION2")).intValue());
+		});
+		
+		UIFormLayout controlsLayout = new UIFormLayout();
+		controlsLayout.setNewlineCount(4);
+		
+		UIPanel controls = new UIPanel();
+		controls.setLayout(controlsLayout);
+		controls.add(new UILabel("Orientation Panel1"));

```

Comment:
Minor, but something like "Orientation of the first panel" might be easier to read.